proto/user-service.proto

```diff
@@ -52,6 +64,20 @@ message UpdateUserResponse {
     User user = 1;
 }
 
+message ValidateTokenResponse {
+    bool valid = 1;
+}
+
+message LoginRequest {
+    string username = 1;
```

Comment:
Oh, I only meant AuthRequest, don't know why I wrote LoginRequest to.
The token is obviously needed but I don't think the username should be.

Shouldn't  the token contain the id of the user that the token belongs to and that is how you do auth checking? I don't think having the username as an additional parameter makes this any more secure then normal jwts.